.gitignore

```diff
@@ -2,3 +2,4 @@
 dist
 node_modules
 examples/example-list.js
+*.swp
```

Comment:
OK, I removed this line and instead created a global gitignore file on my machine like explained in https://help.github.com/articles/ignoring-files/.